src/spec/expectations.cr

```diff
@@ -417,6 +417,26 @@ module Spec
   end
 
   module ObjectExtensions
+    # Validates an expectation and fails the example if it does not match.
+    #
+    # This overload returns a value whose type is restricted to the expected type.. For example:
```

Comment:
A single dot... =)
```suggestion
    # This overload returns a value whose type is restricted to the expected type. For example:
```
And could you add the phrase to the other methods as well?

Then, it should really be fine 🙈